core/src/main/scala/java/text/DecimalFormat.scala

```diff
@@ -1,57 +1,521 @@
 package java.text
 
-class DecimalFormat(private[this] val pattern: String, private[this] var symbols: DecimalFormatSymbols) extends Format {
+import java.math.{RoundingMode, BigDecimal => JavaBigDecimal, BigInteger => JavaBigInteger}
+import java.util.{Currency, Locale}
+import locales.{DecimalFormatUtil, LocaleRegistry, ParsedPattern}
+import scala.math.{max, min}
+
+// The constructor needs a non-localized pattern
+class DecimalFormat(private[this] val pattern: String, private[this] var symbols: DecimalFormatSymbols)
+    extends NumberFormat {
+
   def this(pattern: String) = this(pattern, DecimalFormatSymbols.getInstance())
 
-  //def this() = this("???", DecimalFormatSymbols.getInstance())
+  def this() = {
+    this(
+      LocaleRegistry.ldml(Locale.getDefault).flatMap{ _.numberPatterns.decimalPattern }.getOrElse(???),
```

Comment:
Yeah, unless you thought were getting the localized one, but you forgot to do something else :) - I might crash early here - `the https://pragprog.com/the-pragmatic-programmer/extracts/tips` but in terms of how simple this is, I'm sure the getDefault will throw an error if they weren't installed properly/etc. so I think I'm fine with the default pattern approach